lib/mongoid/criteria/findable.rb

```diff
@@ -32,7 +32,7 @@ def execute_or_raise(ids, multi)
       # @example Find by multiple ids.
       #   criteria.find([ BSON::ObjectId.new, BSON::ObjectId.new ])
       #
-      # @param [ Object | Array<Object> ] *args The ids to search for.
+      # @param [ [ Object | Array<Object> ]... ] *args The id(s) to find.
```

Comment:
This is the correct specification for the other two finds.

Reply:
Noted